corehq/apps/domain/decorators.py

```diff
@@ -43,12 +39,10 @@
     formplayer_auth,
     get_username_and_password_from_request,
 )
-# CCHQ imports
 from corehq.apps.domain.models import Domain, DomainAuditRecordEntry
 from corehq.apps.domain.utils import normalize_domain_name
 from corehq.apps.hqwebapp.signals import clear_login_attempts
 from corehq.apps.users.models import CouchUser
-########################################################################################################
```

Comment:
I vaguely remember reading a rant from Linus Torvalds about stuff like this. He did not love it, and he made his feelings abundantly clear, in his special way. :upside_down_face: